### DISCUSSION TITLE
Phases in an adversary profile

### DISCUSSION THREAD

**Discussion by MRroseB** [mitre/caldera#2334] [Q&A] [answered]
Hi there Caldera users, 
I really like this tool, it's full of cool features and I'm trying to test them all. I'm using the current stable version, so the 3.1.0. I was wondering if it's possible to create phases in an adversary profile. I watched a video on youtube where it shown that it's possible but I was't able to find this feature in the version I'm using. I notice that the GUI in the video was a little bit different so maybe is an older version. I hope you can help me. It would be really useful to create an adversary profile where you can have an attack organized into phases.

**Comment by clenk** [comment, marked as answer]
Hi @MRroseB, you guessed correctly that the video was of an older version. Phases were removed in version 2.6.64. For more information, see https://github.com/mitre/caldera/issues/1597
